Sources/CreateAPI/Generator/Templates.swift

```diff
@@ -421,8 +424,7 @@ final class Templates {
         if let metadata = property.metadata {
             output += comments(for: metadata, name: property.name.rawValue, isProperty: true)
         }
-        let isOptional = property.isOptional && property.defaultValue == nil
-        output += "\(access)\(isReadonly ? "let" : "var") \(property.name): \(property.type)\(isOptional ? "?" : "")"
+        output += "\(access)\(isReadonly ? "let" : "var") \(property.name): \(property.type)\(property.isOptional ? "?" : "")"
```

Comment:
Ahh but it does this because the generated decoder implementation was already using the `defaultValue` as a fallback 🙈 

https://github.com/CreateAPI/CreateAPI/blob/da8730b23dab769414229bb386b3dcc2a77d6e42/Sources/CreateAPI/Generator/Templates.swift#L244-L245

Which is what we said that we should stop doing.

I guess this didn't show up much (at all?) before because it was limited to only booleans. Hmmm, now I'm confused as to what we should do because `includeDefaultValues` is `true` by default.